src/components/GrowlNotification/index.tsx

```diff
@@ -16,15 +19,26 @@ const INACTIVE_POSITION_Y = -255;
 
 const PressableWithoutFeedback = Pressables.PressableWithoutFeedback;
 
-function GrowlNotification(_, ref) {
+function GrowlNotification(_: unknown, ref: ForwardedRef<GrowlRef>) {
```

Comment:
```suggestion
function GrowlNotification(_: never, ref: ForwardedRef<GrowlRef>) {
```

Reply:
Not possible, prop type for JSX components cannot be never as shown in the following screenshot. I'm leaving it as unknown.

<img width="1107" alt="Screenshot 2024-03-12 at 16 29 24" src="https://github.com/Expensify/App/assets/88325488/e1e026d3-1a20-4fce-974f-bd6f3ad1fc0a">